unity/Assets/Scripts/PhysicsRemoteFPSAgentController.cs

```diff
@@ -1513,7 +1513,7 @@ private bool checkForUpDownAngleLimit(string direction, float degrees)
                     return;
                 }
                 Quaternion oldRotation = sop.transform.rotation;
-                sop.transform.rotation = Quaternion.Euler(new Vector3(0.0f, (float) Math.Round(sop.transform.rotation.y + 90f % 360), 0.0f));;
+                sop.transform.rotation = Quaternion.Euler(new Vector3(0.0f, (float) Math.Round((sop.transform.eulerAngles.y + 90f) % 360), 0.0f));;
```

Comment:
Same as above:
> This is unrelated to this PR, but a bug that I discovered that uses `transform.rotation.y` instead of `transform.eulerAngles.y`, which are completely different and results in a silent bug.